prototype_v1/Student-View/modules/course_calendar.R

```diff
@@ -0,0 +1,84 @@
+course_calendar_UI <- function(id){
+  box(width = 12, status = "primary", title = "Course Calendar"
+      , timevisOutput(NS(id,"course_schedule"))
+  )
+}
+
+# Have to rework to use with reactive data 
+course_calendar_server <- function(id, r){
+  moduleServer(id, function(input,output,session){
+    
+    observeEvent(input$course_schedule_selected, {
```

Comment:
Modal pop-up not working
![Screen Shot 2020-10-18 at 10 35 36 PM (2)](https://user-images.githubusercontent.com/52048606/96395403-92292700-1192-11eb-8170-3b5f50740908.png)